scripts/markdown_parse.go

```diff
@@ -0,0 +1,99 @@
+package main
```

Comment:
few nits:
- add a readme
- move to `src/cmd/tools/doctest` or something in the `src/x/` tree

Reply:
add tests too please